pyproject.toml

```diff
@@ -20,12 +20,12 @@ classifiers = [
     'Programming Language :: Python :: 3.12',
 ]
 dependencies = [
-    "boto3<1.24",
+    "boto3>=1.26,<2.0",
     "coloredlogs",
-    "earthaccess~=0.14",
+    "earthaccess>=0.10,<1.0",
     "insitupy~=0.4",
```

Comment:
Let's unpin this for now until we have a final working state. Maybe even do something similar to `snowexsql` and just install the latest from `master/main`